extra/html-extra/HtmlExtension.php

```diff
@@ -28,13 +29,15 @@ public function getFilters(): array
     {
         return [
             new TwigFilter('data_uri', [$this, 'dataUri']),
+            new TwigFilter('attr_merge', 'twig_attr_merge'),
         ];
     }
 
     public function getFunctions(): array
     {
         return [
             new TwigFunction('html_classes', 'twig_html_classes'),
+            new TwigFunction('attr', 'twig_attr', ['needs_environment' => true, 'is_safe' => ['html']]),
```

Comment:
```suggestion
            new TwigFunction('html_attr', 'twig_attr', ['needs_environment' => true, 'is_safe' => ['html']]),
```

Reply:
I guess you prefer the slightly longer `html_attr` for reasons?

Reply:
Because this is specific to the HTML format, right?